R/dashboardSidebar.R

```diff
@@ -117,7 +117,7 @@ dashboardSidebar <- function(..., disable = FALSE, width = NULL) {
     custom_css,
     tags$section(
       class = "sidebar",
-      `data-disable` = if(disable) 1 else NULL,
+      `data-disable` = if(disable) TRUE else NULL,
```

Comment:
IIRC, this should be a `1` because that's what's expected by AdminLTE. Also, `TRUE` is not particularly meaningful in HTML. For example,

```
> tags$section(`foo`=TRUE)
<section foo="TRUE"></section>
```